students/caitlin/drills/strings.js

```diff
@@ -19,19 +19,29 @@
 
 // Create a variable with the name "name"
 // Assign it any relatively human name
+var customerName = "Helen";
 
 // Create a number variable
+var customerNum = 77;
 
 // Create a template literal enthusiastically greeting the above name
+const greeting = `Hello #${customerName}, how are you?`;
```

Comment:
Remove all # signs from file, they're not required for template literals to display variables.